src/types.ts

```diff
@@ -110,6 +108,12 @@ export type AnyNode =
   | swc.MetaProperty
   | swc.MethodProperty
   | swc.Module
+  | swc.ExportNamespaceSpecifier
+  | swc.ExportDefaultSpecifier
+  | swc.NamedExportSpecifier
+  | swc.NamedImportSpecifier
+  | swc.ImportDefaultSpecifier
+  | swc.ImportNamespaceSpecifier
```

Comment:
I believe these are all already there. For example `swc.ExportNamespaceSpecifier` is also on line 73. Double check that these aren't duplicates.

Reply:
I've added `@typescript-eslint/no-duplicate-type-constituents` in order to avoid this